docs/get-started/README.md

```diff
@@ -91,16 +91,52 @@ Now let's make simple CRUD to access our Dataset. For this, you can use the REST
 
 <iframe width="700" height="394" src="https://www.youtube.com/embed/i7v8FOS7_WI" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-In this example, we are using the **JS SDK**. To install it, we need to run:
+<CodeSwitcher :languages="{js:'JavaScript',html:'Browser',py:'Python',bash:'cURL'}">
+<template v-slot:js>
+
 ```
 npm install jexia-sdk-js node-fetch ws --save
 ```
 
-Below you can see an example with all the modules imported into the project. These are optional. If you do not need to access Filesets, Project Users (UMSModule) or real-time events, feel free to skip importing them.  
+</template>
+<template v-slot:html>
+
+``` html
+<html>
+<head>
+     <script src="https://unpkg.com/jexia-sdk-js/bundle/browser.umd.min.js"></script>
+</head>
+```
+
+</template>
+<template v-slot:py>
 
-<CodeSwitcher :languages="{js:'Node', html:'Browser', bash:'cURL'}">
+```
+pip install jexia-sdk
+//or
+easy_install jexia-sdk
+```
+
+</template>
+<template v-slot:bash>
+
+``` bash
+# Environment variables to be set
+export PROJECT_ID=<project_id>
+export API_KEY=<key_here>
+export API_SECRET=<secret_here>
+export TEST_USER=<user_here>
+export TEST_USER_PSW=<password_here>
```

Comment:
Why do we need these variables on this step? As I see they are part of CURL example (lines 230-256 in this file)

Reply:
changed.